phpBB/phpbb/profilefields/type/type_bool.php

```diff
@@ -405,11 +405,14 @@ public function display_options(&$template_vars, &$field_data)
 			$field_data['lang_options'][1] = '';
 		}
 
-		$template_vars = array_merge($template_vars, array(
-			'S_BOOL'					=> true,
-			'L_LANG_OPTIONS_EXPLAIN'	=> $this->user->lang['BOOL_ENTRIES_EXPLAIN'],
-			'FIRST_LANG_OPTION'			=> $field_data['lang_options'][0],
-			'SECOND_LANG_OPTION'		=> $field_data['lang_options'][1],
-		));
+		$doptions = array(
```

Comment:
Reminds me a bit of hungarian notation. Just write out the actual name: `$display_options`